pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/DependencyOrder.java

```diff
@@ -114,7 +119,11 @@ public static ResolvedComponentResult postOrderDependencies(
 
         // Prepare an ordered set of files in post-order.
         for (ResolvedComponentResult d : configurationPostOrderDependencies(configuration)) {
-            files.add(idToFileMap.get(d.getModuleVersion()));
+            if (!idToFileMap.containsKey(d.getModuleVersion())) {
+                logger.lifecycle("***** WARNING Missing resolved artifact for " + d);
```

Comment:
Move this up to warning, if it's actually a warning.

Reply:
Thanks @ethankhall . pr updated